src/OrchardCore.Modules/OrchardCore.Users/Views/UserButtons.cshtml

```diff
@@ -16,6 +20,10 @@
         <a class="btn btn-danger btn-sm" asp-action="Delete" asp-route-id="@user.UserId" data-url-af="RemoveUrl UnsafeUrl" >@T["Delete"]</a>
     }
     <a asp-action="EditPassword" asp-route-id="@user.UserId" class="btn btn-secondary btn-sm">@T["Edit Password"]</a>
+    if (isLockedOut)
```

Comment:
```suggestion
    if (isLockedOut && hasManageUser)
```

Reply:
The code is already inside a `@if (hasManageUser)`